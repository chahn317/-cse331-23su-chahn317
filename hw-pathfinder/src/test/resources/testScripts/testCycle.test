# Test case where there is a cycle

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C

AddEdge graph A B 1.0
AddEdge graph B C 1.0
AddEdge graph C A 1.0

FindPath graph A C
