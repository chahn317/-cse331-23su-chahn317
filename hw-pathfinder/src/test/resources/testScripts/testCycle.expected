# Test case where there is a cycle

created graph graph

added node A to graph
added node B to graph
added node C to graph

added edge 1.000 from A to B in graph
added edge 1.000 from B to C in graph
added edge 1.000 from C to A in graph

path from A to C:
A to B with weight 1.000
B to C with weight 1.000
total cost: 2.000
